### DISCUSSION TITLE
Error Vercel Deploy app React/Node

### DISCUSSION THREAD

**Discussion by tonnelbernardo** [vercel/community#3562] [Help]
### Summary

I'm having an error when deploying my application on vercel, this application is in react and I connect to the MongoDB server using Node, and I get the following errors when deploying on vercel

### Steps to Reproduce

<html>
<body>

Failed to compile.
--
10:40:48.875 |  
10:40:48.876 | [eslint] Failed to load config "react-app" to extend from.
10:40:48.876 | Referenced from: /vercel/path0/.eslintrc.json
10:40:48.876 |  
10:40:48.876 |  
10:40:48.924 | error Command failed with exit code 1.
10:40:48.924 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
10:40:48.948 | Error: Command "yarn run build" exited with 1

</body>
</html>

**Comment by amyegan** [comment]
Do you get any errors in local builds? You can log additional debugging information with [`vercel build --debug`](https://vercel.com/docs/cli/build).
